addons/sale_timesheet/models/account_move.py

```diff
@@ -85,3 +87,6 @@ def _link_timesheets_to_invoice(self, start_date=None, end_date=None):
                     domain = expression.AND([domain, [('date', '<=', end_date)]])
                 timesheets = self.env['account.analytic.line'].sudo().search(domain)
                 timesheets.write({'timesheet_invoice_id': line.move_id.id})
+
+    def _get_range_dates(self):
+        return None, None
```

Comment:
You should provide an explanation that this method aimed to be overriden to provide start and end date corresponding to the order properties. Otherwise it looks like dumb useless code.